index.js

```diff
@@ -1,52 +1,102 @@
-//------------------------Question 1 ---------------------------
-//Is time to create things in the DOM!!. We want you to create a new 'div' Element and append it as a child of the element that has the 'id = a-1' (the element is in index.html file)
-// all right our new element needs style. For that, assign to the new element the className 'square'!. Each time the user clicks on the new element it has to respond changing the displayed text 'x' to 'o'
-
-
-
-
-//resolve // QUESTION 1 here
-
-
-
-
-//------------------------Question 2 ---------------------------
-//Write a JavaScript program to remove items from a dropdown list.
-//In order to create the list we are providing you with the array bellow, those items are the ones you need to add to the dropdown list as the select options
-//you also need to create a button that makes the remove action of the element selected
-// append the new list as a child of the element with the 'id = a-2'
-// append the button as a child of the element with the 'id = a-2'
+//---------------------------------------------------------Question 1 ----------------------------------------------------------------------------------------------------------------
+/* Is time to create things in the DOM!!. We want you to create a new 'div' Element and append it as a child of the element that has the 'id = a-1' 
+(the element is in index.html file) all right our new element needs style. For that, assign to the new element the className 'square'!. 
+Each time the user clicks on the new element it has to respond changing the displayed text 'x' to 'o' 
+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------*/
+
+//Assign 'question1' element to  a 'div' varialbe, Assign 'square' element to a 'square' variable.
+let div = document.getElementById('question1');
+let square = document.getElementsByClassName('square')[0];
+
+// call listen function
+//create new div element 'nDive', add all square syling to the new element
+//Assign string value to the new div, then replace the old div element with the new one!!
+
+div.addEventListener ('click', () => {
+   
+    let nDiv = document.createElement('div');
+    nDiv.classList.add('square')
+    nDiv.innerHTML = "o";         
+    div.replaceChild(nDiv, square); 
```

Comment:
I suggest spending some time clicking around in the inspector in your browser, to see what's actually changed in the DOM by this code when you load the page, and when you click. For example, the div ('question1') doesn't actually have any children at the start, so you don't need to replace its child -- you need to _append_ a child.

Also, see if you can create the square div _once_ -- instead of on every click -- and then on a click, just change its text between x and o. Right now your code is creating a new div on every click, which isn't necessary.